demisto_sdk/commands/common/update_id_set.py

```diff
@@ -1334,6 +1334,47 @@ def process_jobs(file_path: str, packs: Dict[str, Dict], marketplace: str, print
     return result
 
 
+def process_layoutscontainers(file_path: str, packs: Dict[str, Dict], marketplace: str, print_logs: bool) -> Tuple[List, Dict]:
+    """
+    Process a JSON file representing a Layoutcontainer object.
+    Args:
+        file_path: The file path from object field folder.
+        packs: the pack mapping from the ID set.
+        marketplace: the marketplace this id set is designated for.
+        print_logs: Whether to print logs to stdout.
+
+    Returns:
+        a list of Layoutcontainer data.
+    """
+
+    result: List = []
+    excluded_items_from_id_set: dict = {}
+
+    try:
+        if should_skip_item_by_mp(file_path, marketplace, excluded_items_from_id_set, packs=packs, print_logs=print_logs):
+            return result, excluded_items_from_id_set
+
+        if find_type(file_path) != FileType.LAYOUTS_CONTAINER:
+            return result, excluded_items_from_id_set
+
+        # only indicator layouts are supported in marketplace v2.
+        layout_data = get_layoutscontainer_data(file_path, packs=packs)
+        layout_group = next(iter(layout_data.values())).get('group')
+        if marketplace == MarketplaceVersions.MarketplaceV2.value and layout_group == 'incident':
+            add_item_to_exclusion_dict(excluded_items_from_id_set, file_path, layout_data.get('id', ''))
+            return result, excluded_items_from_id_set
+
+        if print_logs:
+            print(f'adding {file_path} to id_set')
+        result.append(layout_data)
+
+    except Exception as exp:  # noqa
```

Comment:
The purpose here is to be on the safe side.
We catch everything, print a log, and raise the original error.
Since we are not handling the error, I think this is ok.